### ISSUE TITLE
Output Results Dock Width

### ISSUE THREAD

**Issue by cocopops17** [nteract/hydrogen#1897]
### :memo: Summary

I would love to be able to define a default width for the "Hydrogen Output Area" pane of the dock when the "View output in the dock by default" option is selected.

### :checkered_flag: Motivation

I find myself manually expanding the width every time I use Hydrogen, which results in imprecise/inconsistent display widths for my editor and impacts the way I typeset my code during every session.

### :runner: Approach

It seems like this feature should be easily implementable via the getPreferredWidth() method for items referenced in atom.workspace.open commands in util.js; however, in my attempts I have not figured out how to successfully implement such an option.

**Comment by cocopops17** [comment]
After some additional digging, I realized I was perhaps looking in the wrong place.  When instead editing Hydrogen\lib\panes\output-area.js, I tried adding the following after the "getAllowedLocations" line:
"getPreferredWidth = () => 1;"

This almost achieved what I wanted: a wider default size for the "Hydrogen Output Area" panel in the right-side dock.  However, it's a still a bit narrower than I would prefer, and increasing the declared value of getPreferredWidth seems to hit a wall past a certain value (e.g. =>1 seems to produce the same width as => 1.25).

I suspect this has something to do with a "max-width" being defined somewhere, but I'm definitely out of my depth at this point.

**Comment by cocopops17** [comment]
After some additional digging, I realized I was looking in the wrong place. By editing "Hydrogen\lib\panes\output-area.js", I was able to add the following after the "getAllowedLocations" line:
"getPreferredWidth = () => atom.workspace.element.offsetWidth*0.45;"
where 0.45 is the percentage of the window width I want to allocate to the output pane.  This seems to work well so far, and might be a nice setting to consider making easily configurable in the package's settings.
